task_02.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""A very basic alarm clock, using raw_input"""
+
+
+DAY = raw_input('What day is it?: ').upper()
+TIME = int(raw_input('What time is it?: '))
+MY_TIME = 600
+
+SNOOZE = (True if DAY[0:3] == ('SAT' or 'SUN') else False and MY_TIME < 600)
```

Comment:
Consider what `DAY[0:3] == ('SAT' or 'SUN')` is really doing -- first, inside parens, it checks the expressions `'SAT'` and `'SUN'` -- since they're just strings and not anything else they check their values as truthy or falsy (eg, `True or True`) since strings that are not empty are truthy. `(True or True)` compiles down to `True`. It then compares that against `DAY[0:3]` as `DAY[0:3] == True` -- also consider that these each return booleans already -- you don't need use a ternary `if` expression here -- they _already_ produce `True|False` by nature of being comparison expressions.